tools/tsdb-symbols/main.go

```diff
@@ -89,7 +89,7 @@ func main() {
 	fmt.Println("Analysis complete in", time.Since(startTime))
 }
 
-func analyseSymbols(_ context.Context, blockDir string, uniqueSymbols map[string]struct{}, uniqueSymbolsPerShard []map[string]struct{}) error {
```

Comment:
ha, how convenient, context was already there, just ignored :)

Reply:
Yep, I did a preparation PR first :)

Reply:
Ah, right! That was your previous PR, I totally forgot.